app/helpers/application_helper.rb

```diff
@@ -417,6 +417,7 @@ def app_metadata
       user_settings: {
         lab_features: Setting[:lab_features],
       },
+      permissions: (User.current.admin? ? Permission.all : User.current.permissions).map { |perm| perm.name },
```

Comment:
`pluck()` allows you to only retrieve that single entry from the DB, so it's probably more efficient to use this:
```suggestion
      permissions: (User.current.admin? ? Permission.all : User.current.permissions).pluck(:name),
```

Reply:
Applied, good idea!